config.py

```diff
@@ -1,4 +1,12 @@
-
 DEBUG = False
 
 NWIS_SITE_SERVICE_ENDPOINT = 'https://waterservices.usgs.gov/nwis/site'
+
+# a list of stream gage ids to be included in the map
+SITE_IDS = []
+
+# coordinates for the map's bounding box
+BOUNDS = ()
+
+# cities to be included on the map
+CITIES = {}
```

Comment:
This geojson may include things like rivers too, so maybe a name like GEOGRAPHICAL_FEATURES.